client/test/execution.test.js

```diff
@@ -414,39 +408,6 @@ describe('Execution', function() {
         compactViewEl.querySelectorAll('.timeline-event.activity.failed').should.have.length(1)
       })
 
-      it('should also populate the timeline with those events', async function() {
-        this.retries(3) // flakey on mocha-chrome but not normal, windowed Chrome
```

Comment:
Found this test was super flaky. Opted to remove than have a breaking test suite